packages/botonic-plugin-contentful/src/nlp/stopwords/README.txt

```diff
@@ -0,0 +1,5 @@
+Default stopwords should not include the following:
```

Comment:
Did you want to put this here? Or it was a misadd??

Reply:
yes, not related to the PR, but since it's only documentation... :-)